monai/fl/utils/constants.py

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+class WeightType:
+    WEIGHTS = "fl_weights_full"
+    WEIGHT_DIFF = "fl_weight_diff"
+
+
+class ExtraItems:
+    ABORT = "fl_abort"
+    MODEL_NAME = "fl_model_name"
+    CLIENT_NAME = "fl_client_name"
+
+
+class FlPhase:
+    IDLE = "fl_idle"
+    TRAIN = "fl_train"
+    PREDICT = "fl_predict"
+    GET_WEIGHTS = "fl_get_weights"
+
+
+class FlStatistics:
+    # TODO: currently uses hardcoded strings from. Might move to MONAI core?
+    #  https://github.com/Project-MONAI/MONAI/blob/df4a7d72e1d231b898f88d92cf981721c49ceaeb/monai/engines/trainer.py#L60
```

Comment:
I would suggest to move them to the engines.

Thanks.

Reply:
Not sure what you mean. Can you help doing this?

Reply:
agree with @Nic-Ma that we may move this to the engine module, let's have follow up PRs... this PR is mainly to move the outcomes of multiple previous discussions/agreements into the codebase